File: tests/07_phases_ompss_2.dg/fortran/success_task_06.f90
! <testinfo>
! test_generator=config/mercurium-ompss-2
! test_nolink=yes
! </testinfo>
PROGRAM P
    IMPLICIT NONE

    INTEGER :: X

    !$OSS TASK INOUT(X) VERIFIED
    !$OSS END TASK

    !$OSS TASK INOUT(X) VERIFIED(X < 5)
    !$OSS END TASK

    !$OSS TASK INOUT(X)
    !$OSS END TASK

    !$OSS TASKWAIT
END PROGRAM P

File: tests/07_phases_ompss_2.dg/c/success_task_06.c
/*
<testinfo>
test_generator=config/mercurium-ompss-2
test_nolink=yes
</testinfo>
*/
int main()
{
    int x;
    #pragma oss task inout(x) verified
    {
    }

    #pragma oss task inout(x) verified(x < 5)
    {
    }

    #pragma oss task inout(x)
    {
    }

    #pragma oss taskwait
}
